README.md

```diff
@@ -22,6 +22,14 @@ Once the cloud resources are provisioned, use the [viya4-deployment](https://git
 SAS Viya 4 in your cloud environment. For more information about SAS Viya 4 requirements and documentation for the deployment
 process, refer to the [SAS&reg; Viya&reg; 4 IT Operations Guide](https://go.documentation.sas.com/doc/en/itopscdc/default/itopswlcm/home.htm).
 
+This project follows the SemVer versioning scheme. Given a version number MAJOR.MINOR.PATCH, we increment the:
```

Comment:
Just a suggestion to add the actual link to semver.org and the summary page to provide a point of reference. The link is : https://semver.org/#summary

Reply:
Done.